### DISCUSSION TITLE
[Question] Pre-training data collection

### DISCUSSION THREAD

**Discussion by Gary3410** [haotian-liu/LLaVA#178] [Q&A]
Thank you for this really great work.  
I have a question about the pre-training dataset collection. In the paper, GPT-4 is employed to generate 158K language-image-command-following samples, and it is pointed out that GPT-4 can generate high-quality instruction-following data. Does this indicate that the data generated by GPT-4 can be directly used for pretraining without artificially filtering low-quality samples？

**Comment by haotian-liu** [comment]
Thanks for your great question.  We filter out some of the responses that contains keywords like "according to the description", "sorry", etc. We'll provide the script of the GPT generation pipeline including these filtering process by the end of this week.  We'll also include this detail in the revision of our paper.  Thanks!

**Comment by lucas-0liveira** [comment]
It seems that the article suggests that GPT-4 has the capability to generate high-quality datasets for instruction following, which raises the question of whether these generated data can be used directly for pre-training without the need to artificially filter out low-quality samples. It would be interesting to delve into the results and methods outlined in the article in more detail to assess the feasibility of this approach.
